diff --git a/git/mdis5/MDISforLinux/TOOLS/HWBUG/linux.c b/git/mdis5/MDISforLinux/TOOLS/HWBUG/linux.c
index c782072..0167bff 100755
--- a/git/mdis5/MDISforLinux/TOOLS/HWBUG/linux.c
+++ b/git/mdis5/MDISforLinux/TOOLS/HWBUG/linux.c
@@ -67,8 +67,8 @@ static char RCSid[]="$Header: /dd2/CVSR/LINUX/TOOLS/HWBUG/linux.c,v 1.6 2010/09/
 #include <termios.h>
 #include <signal.h>
 #include <sys/mman.h>
-#include <asm/io.h>
-#include <asm/errno.h>
+#include <sys/io.h>
+#include <sys/errno.h>
 void os_setterm(), os_restterm(), os_exit();
 static struct termios MyOpts, OrgOpts;
 static int Memdev;
