diff --git a/git/mdis5/INSTALL b/git/mdis5/INSTALL
index 243e3e1..1498fd7 100755
--- a/git/mdis5/INSTALL
+++ b/git/mdis5/INSTALL
@@ -183,30 +183,12 @@ fi
 
 echo " Checking if $MENLINUX_ROOT exists... "
 #
-# Prepare, create /opt/menlinux
+# Prepare, create install directory
 #
 if [ ! -d "$MENLINUX_ROOT" ]; then
    # $MENLINUX_ROOT doesn't exist
-   echo "no"
     
-   echo -n "Checking if $OPT exists... "
-   if [ ! -d "$OPT" ]; then
-      # /opt doesn't exists    
-      echo >&2 "$OPT doesn't exists. Please create and then call $this again"
-      exit 1
-   fi
-   
-
-   # check if /opt writeable
-   if [ ! -w "$OPT" ]; then
-      echo >&2 
-      echo >&2 "*** Can't create directory in $OPT due to insufficient rights."
-      echo >&2 "*** Please login as root and then call $this again"
-      exit 1
-   fi
-   echo "ok"
-   
-   # create MENLINUX
+   # create $MENLINUX_ROOT
    echo -n "Creating directory $MENLINUX_ROOT... "
    do_or_die mkdir -p $MENLINUX_ROOT
    do_or_die chmod 777 $MENLINUX_ROOT
